packages/eas-cli/src/submissions/ios/AppSpecificPasswordSource.ts

```diff
@@ -29,7 +29,7 @@ export async function getAppSpecificPasswordAsync(source: AppSpecificPasswordSou
     const { appSpecificPassword } = await promptAsync({
       name: 'appSpecificPassword',
       message: 'Your Apple app-specific password:',
-      type: 'text',
+      type: 'password',
```

Comment:
Hmm, I'm not sure if this should be a password input. When it's plaintext it's easier to check whether you haven't made any typo when typing in the password.
In my opinion, there is no real benefit from hiding the app-specific password. If you wanted to take over someone's app, knowing only the app-specific password is not enough. You would also have to build and sign a new binary with a distribution certificate and provisioning profile. You can't obtain those without the Apple ID password.

Reply:
I don't anyone will type that, it will be always copy-pasted here

Reply:
But the app-specific password can be used wherever user wants to. We cannot be sure that user uses this password only for app submission.

Reply:
Can an app-specific password be used to generate/manage credentials on Apple Developer Portal?

Reply:
The same password can be used for third party apps:
> App-specific passwords are passwords for your Apple ID that let you sign in to your account and securely access the information you store in iCloud from a third-party app. For example, use app-specific passwords with mail, contacts, and calendar services not provided by Apple.

[Source](https://support.apple.com/en-us/HT204397#:~:text=App%2Dspecific%20passwords%20are%20passwords,services%20not%20provided%20by%20Apple.)

It doesn't allow user to do anything critical with Apple ID account, but in extreme cases the attacker could try to brute-force some 3rd party services with app specific pass.

Reply:
OK! 👍